tests/test_form.py

```diff
@@ -54,3 +71,49 @@ def test_metadata_wrapping(self):
 
         r = form.get_metadata(('cool', 'wow'), TEST_DICT)
```

Comment:
@karlcow maybe this line and the next are a little redundant with the test below for test_get_metadata( )? ~line 103